src/Integration.UnitTests/NewConnectedMode/ConfigurationPersisterTests.cs

```diff
@@ -25,24 +25,25 @@
 using SonarLint.VisualStudio.Core.Binding;
 using SonarLint.VisualStudio.Integration.NewConnectedMode;
 using SonarLint.VisualStudio.Integration.Persistence;
+using SonarLint.VisualStudio.Integration.UnintrusiveBinding;
 
 namespace SonarLint.VisualStudio.Integration.UnitTests
 {
     [TestClass]
     public class ConfigurationPersisterTests
     {
-        private Mock<ISolutionBindingPathProvider> newPathProvider;
```

Comment:
The config persister needs to use the new path provider to save files in the new location.